lib/haml_lint/reporter/disabled_config_reporter.rb

```diff
@@ -23,10 +23,12 @@ def self.available?
     # Create the reporter that will display the report and write the config.
     #
     # @param _log [HamlLint::Logger]
-    def initialize(_log)
-      super
+    def initialize(_log, limit: 15)
+      super(_log)
       @linters_with_lints = Hash.new { |hash, key| hash[key] = [] }
       @linters_lint_count = Hash.new(0)
+      puts "New limit: #{limit}"
```

Comment:
Did you mean to leave this `puts` in here?

Reply:
Oops, no. Will fix it